process/mock/cacherMock.go

```diff
@@ -0,0 +1,95 @@
+package mock
```

Comment:
do we really need this ?

Reply:
need this mock for TestScrsPreprocessor_RestoreTxBlockIntoPools test